### PR TITLE
Addressing TAG review issue#133, 134, and 137

### PR DESCRIPTION
**Pull request by wchao1115**
- #133 Move Relu into the unary operator section.
- #134 Clarify in the descriptions of BatchNormalization and InstanceNormalization
- #137 Clarify in the descriptions of Reshape and Squeeze about the content-neutral nature of the operations.

***
<a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/pull/144.html" title="Last updated on Feb 20, 2021, 4:53 AM UTC (44870de)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/144/356de2d...44870de.html" title="Last updated on Feb 20, 2021, 4:53 AM UTC (44870de)">Diff</a>

### CODE REVIEW

**Comment by wchao1115** [comment]
https://github.com/w3ctag/design-reviews/issues/570#issuecomment-768875996

File: index.bs [hunk 1]
```diff
@@ -1128,35 +1130,6 @@ partial interface ModelBuilder {
...
-
-    <div class="note">
-    The behavior of this operation can be generically emulated from the usage of
-    other operations as follow. However, user agents typically have a more
-    efficient implementation for it, therefore its usage is encouraged from the
-    performance standpoint.
-    <pre highlight="js">
-    return builder.max(builder.constant(0), x);
-    </pre>
-    </div>
```
Comment (huningxin) [inline index.bs:1157]: Do we need to keep the sample code that decomposes `relu`?

**Review by anssiko — APPROVED** [review]

**Comment by anssiko** [comment]
@cynthia do you prefer to be explicitly requested for review for these PRs that address TAG feedback?

**Comment by cynthia** [comment]
@anssiko Not necessarily, but I would like to know which points are acknowledged as valid issues. (and how the group plans to address it, I have subscribed to some of them but haven't seen much aside from the issue being opened)

**Comment by anssiko** [comment]
@cynthia the proposed plan of record is to open PRs to address each valid [webnn tag-tracker issue](https://github.com/webmachinelearning/webnn/labels/tag-tracker) similarly to this PR. I’m hearing we won’t request/block on your PR review, but rather post a summary of all the changes to the [master TAG review issue](https://github.com/w3ctag/design-reviews/issues/570) once done with all if it, so the TAG can have a group discussion and see if you’re satisfied with our response.

If there are some specific clarifying questions we’ll ask you about them explicitly in respective issues or PRs. For thorny issues, if any, we may invite you to join our call.

@cynthia sounds good? Thanks for your insights!

File: index.bs (hunk 1 above)
Comment (wchao1115) [reply index.bs:1157]: Fixed.

**Review by huningxin — APPROVED** [review]
Looks good. Thanks @wchao1115 !

**Comment by cynthia** [comment]
> post a summary of all the changes to the master TAG review issue once done

Sounds good to me. I'll subscribe myself to some of the more contentious issues - feel free to mention me if you need input on specific issues.

**Pull request merged**